### DISCUSSION TITLE
How do i make swiper uninteractivable, i want that user cannot change slide in any way possible i want to autoplay hande all the changing

### DISCUSSION THREAD

**Discussion by picarica** [nolimits4web/swiper#4718] [Q&A] [answered]
so i wanted to disable any interaction to slider by user i still want user to be able to highlight and click on stuff in the slider i just dont want him to change slider or disable autoplay in any way i  thought noSwiping: true would solve this but the issue still persist, i still can change slide using my mouse, i dont want to be changeble by mouse by finger by anything is this possible ?
here is my current setup

const swiper = new Swiper(\'.swiper-container\', {
	centeredSlides: true,
	slidesPerView: 1,
	loop:true,
	autoplay: {
		delay: 5000,
	},
	disableOnInteraction:false,
	speed:1500,
	noSwiping: true,
	
	pagination: {
          el: ".swiper-pagination",
          type: "progressbar",
        },
});

**Comment by nolimits4web** [comment, marked as answer]
There is a `allowTouchMove` parameter for that, set it to `false`

**Reply by picarica** [reply]
yes this is it, sorry i didnt noticed it, i was lookin on that page for a while
